go-controller/pkg/node/node.go

```diff
@@ -625,22 +624,12 @@ func (n *OvnNode) validateVTEPInterfaceMTU() error {
 		requiredMTU = config.Default.MTU + types.GeneveHeaderLengthIPv6
 	}
 
-	// check if node needs to be tainted
 	if mtu < requiredMTU {
-		klog.V(2).Infof("MTU (%d) of network interface %s is not big enough to deal with Geneve "+
-			"header overhead (sum %d). Tainting node with %v...", mtu, interfaceName,
-			requiredMTU, tooSmallMTUTaint)
-
-		return retry.RetryOnConflict(retry.DefaultRetry, func() error {
-			return n.Kube.SetTaintOnNode(n.name, tooSmallMTUTaint)
-		})
+		return fmt.Errorf("interface MTU (%d) is too small for specified overlay MTU (%d)", mtu, requiredMTU)
 	}
 	klog.V(2).Infof("MTU (%d) of network interface %s is big enough to deal with Geneve header overhead (sum %d). "+
 		"Making sure node is not tainted with %v...", mtu, interfaceName, requiredMTU, tooSmallMTUTaint)
```

Comment:
I think it would be better to clean up references to tainting in this log and to get rid of `tooSmallMTUTaint`